modules/ensemble/lib/widget/confirmation_input.dart

```diff
@@ -244,24 +246,21 @@ class ConfirmationInputState extends framework.WidgetState<ConfirmationInput>
   }
 }
 
-String _validatePinTypeCustom(String? value) {
-    if (value == null || value.length != 1) {
-      return "*";
+dynamic _validatePinTypeCustom(dynamic value) {
+  if ( value is String ) {
+    if (  value.length != 1) {
+      return "*"; // Default symbol if string length is not 1
     }
     return value;
+  } else if (value is Map && value['icon'] != null) {
```

Comment:
what if it's YamlMap? I believe YamlMap is Map but please test. also test when the value is being set in js

Reply:
Yup testes with multiple inputs, added js inputs in description as well.